spectroscopy/code_src/mast_functions.py

```diff
@@ -96,6 +96,8 @@ def JWST_get_spec_helper(sample_table, search_radius_arcsec, datadir, verbose,
 
         # Query results
         search_coords = stab["coord"]
+        # [FIXME] Next code line raises a warning when stab['label'] == 'NGC3049'.
+        # 'WARNING: NoResultsWarning: Query returned no results. [astroquery.mast.discovery_portal]'
```

Comment:
We can filter these out as it's expected that not all input coordinate/objects will have coverage. But I would do that filter very explicitly with detailed comments, and here and only within this scope rather than in the notebook. As I would love to avoid teaching users to ignore warnings by default.